Casks/t/tentacle-sync-studio.rb

```diff
@@ -2,14 +2,14 @@
   version "1.34"
   sha256 "e2ec89770d20a892575e870c35d5bf5bdd5e4cdce5a82c4549159554ce6fad3d"
 
-  url "https://tentaclesync.com/files/downloads/ttsyncstudio-v#{version.dots_to_underscores}.dmg"
+  url "https://cms.tentaclesync.com/assets/downloads/download-files/ttsyncstudio-v#{version.dots_to_underscores}.dmg"
   name "Tentacle Sync Studio"
   desc "Automatically synchronise video and audio via timecode"
   homepage "https://tentaclesync.com/"
 
   livecheck do
-    url "https://tentaclesync.zendesk.com/api/v2/help_center/en-us/articles/115003866805"
-    regex(%r{href=.*?/ttsyncstudio-v?(\d+(?:[._-]\d+)+)\.dmg}i)
+    url "https://tentaclesync.com/api/collections/downloads/entries?filter[site]=en&"
```

Comment:
```suggestion
    url "https://tentaclesync.com/api/collections/downloads/entries"
```

This filter does not appear to affect the output.